clang/tools/checked-c-convert/IterativeItypeHelper.h

```diff
@@ -0,0 +1,39 @@
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+// This file contains the helper classes that are used for constraint solving
+// based on iterative itype refinement.
+//===----------------------------------------------------------------------===//
+#ifndef _ITYPECONSTRAINTDETECTOR_H
+#define _ITYPECONSTRAINTDETECTOR_H
+
+#include "clang/AST/Decl.h"
+#include "clang/AST/Stmt.h"
+#include "clang/AST/ASTContext.h"
+#include "clang/Rewrite/Core/Rewriter.h"
+
+#include "ProgramInfo.h"
+
+using namespace clang;
+
+// this method resets the constraint graph by removing
```

Comment:
Please make the capitalization of comments uniform. Please start comments with a capital letter and end them with a period.